pkg/cmd/infra/router/template.go

```diff
@@ -438,3 +492,21 @@ func (o *TemplateRouterOptions) Run() error {
 
 	select {}
 }
+
+// blueprintRoutes returns all the routes in the blueprint namespace.
+func (o *TemplateRouterOptions) blueprintRoutes(routeclient *routeinternalclientset.Clientset) ([]*routeapi.Route, error) {
+	blueprints := make([]*routeapi.Route, 0)
+	if len(o.BlueprintRouteNamespace) == 0 {
```

Comment:
I don't like an entire namespace of route blueprints.  Require them to be identified with a label or associated metadata within a specific namespace, or require them to be specified by name directly.

Whole namespaces devoted to this is pretty ugly, although the idea of pulling it from the route is good.

Reply:
ok will do.